src/components/ProfilePage/Bio.jsx

```diff
@@ -61,16 +61,16 @@ class Bio extends React.Component {
                   type="textarea"
                   id={formId}
                   name={formId}
-                  value={value}
+                  value={bio || ''}
                   invalid={error != null}
                   onChange={this.handleChange}
                 />
                 <FormFeedback>{error}</FormFeedback>
               </FormGroup>
               <FormControls
-                formId={formId}
+                visibilityId="visibilityBio"
```

Comment:
It's a bit odd that the visibility dropdown is in a class called "FormControls" - it's a bit specific to this profile page's layout.  That said, because it's there, we need to pass in some info for it so it can be given a meaningful ID.

Reply:
would you prefer passing in the whole <VisibilitySelect /> in as a prop to FormControls?
`visibilitySelect={<VisibilitySelect id="visibilityBio" />}`

Reply:
I think it would make more sense to pull it out completely.  It renders on a row above the buttons - there's not much reason it needs to be in there, I'd think.  Using Bio as an example, it's a form element that's effectively a peer of the textarea input... having it bundled with the form controls themselves seems odd, you know?